cudamapper/src/utils.cpp

```diff
@@ -161,8 +161,8 @@ static inline void encode_seq(bam1_t* const alignment, const std::string& seq)
     // encode bases as nibbles and write them into the alignment struct 2 bases at a time
     // 4 high bits are first base, 4 low bits are second base
     uint8_t* bam_seq_start = bam_get_seq(alignment);
-    int i                  = 0;
-    for (i; (i + 1) < len; i += 2)
+    uint32_t i             = 0;
```

Comment:
Again, we tend to use signed values whenever possible (as suggested by ahehn-nv), so you can use https://github.com/clara-parabricks/GenomeWorks/blob/df8b1f38e48e2aacc13967cfa863f036c1dd37b4/common/base/include/claraparabricks/genomeworks/utils/signed_integer_utils.hpp#L31